### PR TITLE
Test 3

### PR DESCRIPTION
**Pull request by isharac**
org.wso2.am.integration.tests.header.CORSHeadersTestCase
org.wso2.am.integration.tests.header.CustomHeaderTestCase
org.wso2.am.integration.tests.login.EmailUserNameLoginTestCase
org.wso2.am.integration.tests.other.APISearchAPIByTagTestCase
org.wso2.am.integration.tests.header.CORSAccessControlAllowCredentialsHeaderTestCase

### CODE REVIEW

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/header/CORSAccessControlAllowCredentialsHeaderTestCase.java [hunk 1]
```diff
@@ -66,7 +76,7 @@
 public class CORSAccessControlAllowCredentialsHeaderTestCase extends APIManagerLifecycleBaseTest {
 
     private String publisherURLHttp;
-    private APIPublisherRestClient apiPublisher;
+//    private APIPublisherRestClient apiPublisher;
```
Comment (mushthaq33) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/header/CORSAccessControlAllowCredentialsHeaderTestCase.java]: Lets remove this line.

File: modules/integration/tests-integration/tests-backend/src/test/resources/testng.xml [hunk 2]
```diff
@@ -31,6 +31,12 @@
             <!--<class name="org.wso2.am.integration.tests.jwt.JWTRevocationTestCase"/>-->
         </classes>
     </test>
+    <test name="apim-integration-tests-api-lifecycle" preserve-order="true" parallel="false">
```
Comment (mushthaq33) [inline modules/integration/tests-integration/tests-backend/src/test/resources/testng.xml]: Lets remove the testng commit for now

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/header/CORSAccessControlAllowCredentialsHeaderTestCase.java (hunk 1 above)
Comment (isharac) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/header/CORSAccessControlAllowCredentialsHeaderTestCase.java]: fixed

File: modules/integration/tests-integration/tests-backend/src/test/resources/testng.xml (hunk 2 above)
Comment (isharac) [reply modules/integration/tests-integration/tests-backend/src/test/resources/testng.xml]: done

**Pull request merged**